2024-08-18 04:11:56,014 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-08-18 04:11:56,014 INFO: [33mPress CTRL+C to quit[0m
2024-08-18 04:11:56,014 INFO:  * Restarting with stat
2024-08-18 04:11:56,230 WARNING:  * Debugger is active!
2024-08-18 04:11:56,236 INFO:  * Debugger PIN: 617-080-234
2024-08-18 04:12:14,876 INFO: 127.0.0.1 - - [18/Aug/2024 04:12:14] "[35m[1mPOST /users HTTP/1.1[0m" 201 -
2024-08-18 04:12:23,366 INFO: 127.0.0.1 - - [18/Aug/2024 04:12:23] "GET /users HTTP/1.1" 200 -
2024-08-18 04:12:29,854 ERROR: Error: Object of type function is not JSON serializable
Traceback (most recent call last):
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 493, in wrapper
    return self.make_response(data, code, headers=headers)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 522, in make_response
    resp = self.representations[mediatype](data, *args, **kwargs)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/representations/json.py", line 21, in output_json
    dumped = dumps(data, **settings) + "\n"
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 201, in encode
    chunks = list(chunks)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 438, in _iterencode
    o = _default(o)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type function is not JSON serializable
2024-08-18 04:12:29,859 INFO: 127.0.0.1 - - [18/Aug/2024 04:12:29] "[35m[1mGET /my-transactions HTTP/1.1[0m" 500 -
2024-08-18 04:12:31,902 ERROR: Error: Object of type function is not JSON serializable
Traceback (most recent call last):
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 493, in wrapper
    return self.make_response(data, code, headers=headers)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 522, in make_response
    resp = self.representations[mediatype](data, *args, **kwargs)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/representations/json.py", line 21, in output_json
    dumped = dumps(data, **settings) + "\n"
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 201, in encode
    chunks = list(chunks)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 438, in _iterencode
    o = _default(o)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type function is not JSON serializable
2024-08-18 04:12:31,903 INFO: 127.0.0.1 - - [18/Aug/2024 04:12:31] "[35m[1mGET /my-transactions HTTP/1.1[0m" 500 -
2024-08-18 04:12:39,735 ERROR: Error: Object of type function is not JSON serializable
Traceback (most recent call last):
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 493, in wrapper
    return self.make_response(data, code, headers=headers)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 522, in make_response
    resp = self.representations[mediatype](data, *args, **kwargs)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/representations/json.py", line 21, in output_json
    dumped = dumps(data, **settings) + "\n"
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 201, in encode
    chunks = list(chunks)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 438, in _iterencode
    o = _default(o)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type function is not JSON serializable
2024-08-18 04:12:39,736 INFO: 127.0.0.1 - - [18/Aug/2024 04:12:39] "[35m[1mGET /my-transactions HTTP/1.1[0m" 500 -
2024-08-18 04:12:51,325 ERROR: Error: Object of type function is not JSON serializable
Traceback (most recent call last):
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 493, in wrapper
    return self.make_response(data, code, headers=headers)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 522, in make_response
    resp = self.representations[mediatype](data, *args, **kwargs)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/representations/json.py", line 21, in output_json
    dumped = dumps(data, **settings) + "\n"
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 201, in encode
    chunks = list(chunks)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 438, in _iterencode
    o = _default(o)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type function is not JSON serializable
2024-08-18 04:12:51,326 INFO: 127.0.0.1 - - [18/Aug/2024 04:12:51] "[35m[1mGET /my-transactions HTTP/1.1[0m" 500 -
2024-08-18 04:13:09,019 ERROR: Error: Object of type function is not JSON serializable
Traceback (most recent call last):
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 493, in wrapper
    return self.make_response(data, code, headers=headers)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 522, in make_response
    resp = self.representations[mediatype](data, *args, **kwargs)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/representations/json.py", line 21, in output_json
    dumped = dumps(data, **settings) + "\n"
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 201, in encode
    chunks = list(chunks)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 438, in _iterencode
    o = _default(o)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type function is not JSON serializable
2024-08-18 04:13:09,021 INFO: 127.0.0.1 - - [18/Aug/2024 04:13:09] "[35m[1mGET /my-transactions HTTP/1.1[0m" 500 -
2024-08-18 04:18:23,654 INFO:  * Detected change in '/Users/emmanuel/Development/projects/PFapp/server/models.py', reloading
2024-08-18 04:18:23,705 INFO:  * Restarting with stat
2024-08-18 04:18:24,134 WARNING:  * Debugger is active!
2024-08-18 04:18:24,140 INFO:  * Debugger PIN: 617-080-234
2024-08-18 04:18:47,355 INFO:  * Detected change in '/Users/emmanuel/Development/projects/PFapp/server/models.py', reloading
2024-08-18 04:18:47,408 INFO:  * Restarting with stat
2024-08-18 04:18:47,724 WARNING:  * Debugger is active!
2024-08-18 04:18:47,729 INFO:  * Debugger PIN: 617-080-234
2024-08-18 04:18:52,965 ERROR: Error: Object of type function is not JSON serializable
Traceback (most recent call last):
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 493, in wrapper
    return self.make_response(data, code, headers=headers)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 522, in make_response
    resp = self.representations[mediatype](data, *args, **kwargs)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/representations/json.py", line 21, in output_json
    dumped = dumps(data, **settings) + "\n"
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 201, in encode
    chunks = list(chunks)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 438, in _iterencode
    o = _default(o)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type function is not JSON serializable
2024-08-18 04:18:52,970 INFO: 127.0.0.1 - - [18/Aug/2024 04:18:52] "[35m[1mGET /my-transactions HTTP/1.1[0m" 500 -
2024-08-18 04:18:54,200 ERROR: Error: Object of type function is not JSON serializable
Traceback (most recent call last):
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 493, in wrapper
    return self.make_response(data, code, headers=headers)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 522, in make_response
    resp = self.representations[mediatype](data, *args, **kwargs)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/representations/json.py", line 21, in output_json
    dumped = dumps(data, **settings) + "\n"
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 201, in encode
    chunks = list(chunks)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 438, in _iterencode
    o = _default(o)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type function is not JSON serializable
2024-08-18 04:18:54,202 INFO: 127.0.0.1 - - [18/Aug/2024 04:18:54] "[35m[1mGET /my-transactions HTTP/1.1[0m" 500 -
2024-08-18 04:19:01,867 INFO:  * Detected change in '/Users/emmanuel/Development/projects/PFapp/server/models.py', reloading
2024-08-18 04:19:01,928 INFO:  * Restarting with stat
2024-08-18 04:19:02,281 WARNING:  * Debugger is active!
2024-08-18 04:19:02,286 INFO:  * Debugger PIN: 617-080-234
2024-08-18 04:19:09,349 INFO:  * Detected change in '/Users/emmanuel/Development/projects/PFapp/server/models.py', reloading
2024-08-18 04:19:09,403 INFO:  * Restarting with stat
2024-08-18 04:19:09,764 WARNING:  * Debugger is active!
2024-08-18 04:19:09,779 INFO:  * Debugger PIN: 617-080-234
2024-08-18 04:19:11,053 ERROR: Error: Object of type function is not JSON serializable
Traceback (most recent call last):
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 493, in wrapper
    return self.make_response(data, code, headers=headers)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 522, in make_response
    resp = self.representations[mediatype](data, *args, **kwargs)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/representations/json.py", line 21, in output_json
    dumped = dumps(data, **settings) + "\n"
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 201, in encode
    chunks = list(chunks)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 438, in _iterencode
    o = _default(o)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type function is not JSON serializable
2024-08-18 04:19:11,058 INFO: 127.0.0.1 - - [18/Aug/2024 04:19:11] "[35m[1mGET /my-transactions HTTP/1.1[0m" 500 -
2024-08-18 04:19:11,888 ERROR: Error: Object of type function is not JSON serializable
Traceback (most recent call last):
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 493, in wrapper
    return self.make_response(data, code, headers=headers)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 522, in make_response
    resp = self.representations[mediatype](data, *args, **kwargs)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/representations/json.py", line 21, in output_json
    dumped = dumps(data, **settings) + "\n"
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 201, in encode
    chunks = list(chunks)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 438, in _iterencode
    o = _default(o)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type function is not JSON serializable
2024-08-18 04:19:11,890 INFO: 127.0.0.1 - - [18/Aug/2024 04:19:11] "[35m[1mGET /my-transactions HTTP/1.1[0m" 500 -
2024-08-18 04:19:20,873 INFO:  * Detected change in '/Users/emmanuel/Development/projects/PFapp/server/models.py', reloading
2024-08-18 04:19:20,938 INFO:  * Restarting with stat
2024-08-18 04:19:21,254 WARNING:  * Debugger is active!
2024-08-18 04:19:21,259 INFO:  * Debugger PIN: 617-080-234
2024-08-18 04:19:24,286 INFO:  * Detected change in '/Users/emmanuel/Development/projects/PFapp/server/models.py', reloading
2024-08-18 04:19:24,323 INFO:  * Restarting with stat
2024-08-18 04:19:24,544 WARNING:  * Debugger is active!
2024-08-18 04:19:24,548 INFO:  * Debugger PIN: 617-080-234
2024-08-18 04:22:18,069 INFO:  * Detected change in '/Users/emmanuel/Development/projects/PFapp/server/models.py', reloading
2024-08-18 04:22:18,128 INFO:  * Restarting with stat
2024-08-18 04:22:18,473 WARNING:  * Debugger is active!
2024-08-18 04:22:18,479 INFO:  * Debugger PIN: 617-080-234
2024-08-18 04:22:22,515 INFO:  * Detected change in '/Users/emmanuel/Development/projects/PFapp/server/models.py', reloading
2024-08-18 04:22:22,550 INFO:  * Restarting with stat
2024-08-18 04:22:22,779 WARNING:  * Debugger is active!
2024-08-18 04:22:22,783 INFO:  * Debugger PIN: 617-080-234
2024-08-18 04:22:27,821 INFO:  * Detected change in '/Users/emmanuel/Development/projects/PFapp/server/models.py', reloading
2024-08-18 04:22:27,902 INFO:  * Restarting with stat
2024-08-18 04:22:28,186 WARNING:  * Debugger is active!
2024-08-18 04:22:28,192 INFO:  * Debugger PIN: 617-080-234
2024-08-18 04:23:09,534 INFO:  * Detected change in '/Users/emmanuel/Development/projects/PFapp/server/models.py', reloading
2024-08-18 04:23:09,586 INFO:  * Restarting with stat
2024-08-18 04:25:01,581 INFO: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5555
2024-08-18 04:25:01,582 INFO: [33mPress CTRL+C to quit[0m
2024-08-18 04:25:01,582 INFO:  * Restarting with stat
2024-08-18 04:25:01,798 WARNING:  * Debugger is active!
2024-08-18 04:25:01,804 INFO:  * Debugger PIN: 617-080-234
2024-08-18 04:25:05,015 ERROR: Error: Object of type function is not JSON serializable
Traceback (most recent call last):
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 493, in wrapper
    return self.make_response(data, code, headers=headers)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/__init__.py", line 522, in make_response
    resp = self.representations[mediatype](data, *args, **kwargs)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/site-packages/flask_restful/representations/json.py", line 21, in output_json
    dumped = dumps(data, **settings) + "\n"
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 201, in encode
    chunks = list(chunks)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 438, in _iterencode
    o = _default(o)
  File "/Users/emmanuel/.pyenv/versions/3.8.13/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type function is not JSON serializable
2024-08-18 04:25:05,019 INFO: 127.0.0.1 - - [18/Aug/2024 04:25:05] "[35m[1mGET /my-transactions HTTP/1.1[0m" 500 -
2024-08-18 04:25:18,933 INFO:  * Detected change in '/Users/emmanuel/Development/projects/PFapp/server/models.py', reloading
2024-08-18 04:25:18,972 INFO:  * Restarting with stat
